include/stdexec/execution.hpp

```diff
@@ -3917,12 +3917,12 @@ namespace stdexec {
   // [execution.senders.adaptors.stopped_as_optional]
   // [execution.senders.adaptors.stopped_as_error]
   namespace __stopped_as_xxx {
-    template <class _SenderId, class _ReceiverId>
+    template <class _CvrefSenderId, class _ReceiverId>
       struct __operation;
 
-    template <class _SenderId, class _ReceiverId>
+    template <class _CvrefSenderId, class _ReceiverId>
       struct __receiver {
-        using _Sender = stdexec::__t<_SenderId>;
+        using _Sender = stdexec::__t<_CvrefSenderId>;
```

Comment:
I have an issue to remove the dependency on the sender type here. Just sayin', no action needed for this PR.